package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gasbuddy/configured-swagger-client",
-  "version": "5.4.1",
+  "version": "5.4.2",
```

Comment:
This is probably a minor version change not a patch. It's changing a behavior not fixing a bug...